Sources/ProjectAutomation/Tuist.swift

```diff
@@ -34,7 +34,7 @@ public enum Tuist {
             var arguments = [
                 "tuist",
                 "graph",
-                "--format", "json",
+                "--format", "--legacyJSON",
```

Comment:
```suggestion
                "--format", "legacyJSON",
```

My bad, it's without the extra hyphens.